pkg/virtctl/vnc/vnc.go

```diff
@@ -112,12 +121,9 @@ func (o *VNC) Run(cmd *cobra.Command, args []string) error {
 		viewResChan <- err
 	}()
 
-	select {
-	case err = <-viewResChan:
-		if err != nil {
-			return err
-		}
-	}
+	glog.Infof("connection timeout: %v", LISTEN_TIMEOUT)
+	// exit early if spawning remote-viewer fails
+	ln.SetDeadline(time.Now().Add(LISTEN_TIMEOUT))
```

Comment:
@rmohr I did this in parallel to your review, before to see your comments :) I can either keep this or do like you suggested, no strong feelings. Your call.